### PR TITLE
Add form settings test

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**

### CODE REVIEW

File: app/instrumentation-tests/src/org/commcare/androidTests/FormSettingsTest.kt [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        val fontValues = context.resources.getStringArray(R.array.font_size_entry_values)
+
+        // Change font size to extra small and confirm if the question text size changes
+        val extraSmallFontIndex = fontNames.size - 1
+        changeFontSize(fontNames[extraSmallFontIndex])
+        onView(withText(startsWith("The following questions")))
+                .check(matches(CustomMatchers.withFontSize(fontValues[extraSmallFontIndex].toFloat())))
+
+        val mediumFontIndex = 2
+        changeFontSize(fontNames[mediumFontIndex])
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/androidTests/FormSettingsTest.kt:122]: Should we have a similar check as L118 for checking if the size changed to medium or not ?

File: app/instrumentation-tests/src/org/commcare/utils/CustomMatchers.kt [hunk 2]
```diff
@@ -105,4 +107,22 @@ object CustomMatchers {
         }
     }
 
+    fun withFontSize(expectedSize: Float): Matcher<View> {
```
Comment (shubham1g5) [inline app/instrumentation-tests/src/org/commcare/utils/CustomMatchers.kt:110]: nice work implementing this.

File: app/instrumentation-tests/src/org/commcare/androidTests/FormSettingsTest.kt (hunk 1 above)
Comment (ShivamPokhriyal) [reply app/instrumentation-tests/src/org/commcare/androidTests/FormSettingsTest.kt:122]: Sounds good, will add the check for it.

**Review by shubham1g5 — APPROVED** [review]

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462055/)

**Pull request merged**